server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -205,7 +205,7 @@ private boolean prepareAndUpload(Long primaryTerm, Long generation) throws IOExc
                     readers.add(reader);
                     copyCheckpointTo(location.resolve(getCommitCheckpointFileName(current.getGeneration())));
                     if (closed.get() == false) {
-                        logger.trace("Creating new writer for gen: [{}]", current.getGeneration() + 1);
+                        logger.trace("{} Creating new writer for gen: [{}]", shardId, current.getGeneration() + 1);
```

Comment:
Why are we not creating logger with ShardId information in the RemoteFsTranslog ctor? 
RemoteFsTranslog has TranslogConfig which holds the reference to shardId.
https://github.com/opensearch-project/OpenSearch/blob/11f46994a6a0e4fa510fb878891446ccb007fbf4/server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java#L73-L83

https://github.com/opensearch-project/OpenSearch/blob/11f46994a6a0e4fa510fb878891446ccb007fbf4/server/src/main/java/org/opensearch/index/translog/TranslogConfig.java#L89-L91